docs/user_guide/models/destvi.rst

```diff
@@ -0,0 +1,206 @@
+model, along a spatial gene expression matrix :math:`Y` with :math:`S` spots and :math:`G` genes.
+Optionally, the user can specify the number of components used for the mixture model underlying the
+emprical prior.
+
+
+Generative process
+==================
+
+scLVM
+-----
+
+For cell :math:`n`, the scLVM assumes observed discrete cell type labels :math:`c_n` and models
+continuous covariates :math:`\gamma_n` of dimension :math:`d` to explain variation in gene expression within a cell type.
+The scLVM posits that the observed UMI counts for cell :math:`n` are generated by the following process:
+
+.. math::
+    :nowrap:
+
+    \begin{align}
+        \gamma_n &\sim \textrm{Normal}(0, I) \tag{1} \\
+        x_{ng} &\sim \textrm{NegativeBinomial}(l_nf^g(c_n, \gamma_n), p_g) \tag{2} \\
+    \end{align}
+
+where :math:`l_n` is the library size, :math:`f` is a two-layer neural network which outputs a :math:`G`
+dimensional vector, and :math:`p_g` is the rate parameter of the negative binomial distribution for
+a given gene :math:`g`. Note that we are using the standard rate-shape parametrization of the
+negative binomial here, rather than the mean-dispersian parametrization used in :doc:`/user_guide/models/scvi`.
+
+The latent variables for the scLVM, along with their description are summarized in the following table:
+
+.. list-table::
+   :widths: 20 90 15
+   :header-rows: 1
+
+   * - Latent variable
+     - Description
+     - Code variable (if different)
+   * - :math:`\gamma_n \in \mathbb{R}^d`
+     - Low-dimensional representation of sub-cell-type covariates.
+     - ``z``
+   * - :math:`p_g \in \mathbb{R}^{+}`
+     - Rate parameter for negative binomial distribution.
+     - ``px_r``
+
+stLVM
+-----
+
+For the stLVM, we also model the expression counts with a :math:`\mathrm{NegativeBinomial}`. However,
+for spatial data, we assume that each spot :math:`s` has expression :math:`x_s` composed of gene expression
+counts of :math:`C(s)` individual cells. We assume that cell :math:`n` in spot :math:`s` is generated from
+the latent variables :math:`(c_{ns}, \gamma_{ns})` by the following process:
+
+.. math::
+    :nowrap:
+
+    \begin{align}
+        x_{sg} &\sim \mathrm{NegativeBinomial}(l_s\alpha_g\sum_{n=1}^{C(s)}f^g(c_{ns}, \gamma_{ns}), p_g) \tag{3} \\
+    \end{align}
+
+where :math:`l_s` is the library size and :math:`\alpha_g` is a correction term for
+difference in experimental assays. Like the scLVM, :math:`f` is a decoder neural network, and
+:math:`p_g` is the rate parameter for the negative binomial distribution. So, we can model
+the cell type abundances :math:`\beta_{sc}` per spot rather than :math:`C(n)` and the
+corresponding cell types for each cell, we parametrize the sum from :math:`(3)` over cell types.
+
+.. math::
+    :nowrap:
+
+    \begin{align}
+        x_{sg} &\sim \mathrm{NegativeBinomial}(l_s\alpha_g\sum_{c=1}^{C}\beta_{sc}f^g(c, \gamma_s^c), p_g) \tag{4} \\
+    \end{align}
```

Comment:
I would include how gamma is generated (vampprior)

Reply:
yes!